tests/e2e/cypress/cypress/integration/editor/ci-po/create-ci-po.spec.js

```diff
@@ -0,0 +1,85 @@
+/// <reference types="Cypress" />
```

Comment:
Is it normal the dash between `ci` and `po` into the name file ?

Reply:
Yes